### DISCUSSION TITLE
Silent build

### DISCUSSION THREAD

**Discussion by fabriziogiordano** [withastro/roadmap#400] [Proposal]
Would it be possible to add an option to silent the build process?

<img width="355" alt="image" src="https://user-images.githubusercontent.com/109390/204475075-3ee855de-72ef-426a-9f0c-8a3c4c93b63e.png">

and might be just show the process on the main pages/routes?

**Comment by jasikpark** [comment]
What does `--silent` do for you?

https://docs.astro.build/en/reference/cli-reference/#--silent

**Comment by fabriziogiordano** [comment]
I wonder if there should be a mid type of log.
Just show progress without necessarily show each file.

Thank you @jasikpark !!
